change/@azure-msal-angular-7c84ff5a-7773-4dde-a551-3665adb6e468.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

I think this can be a prerelease change
```suggestion
  "type": "prerelease",
```

---

Yes. Makes sense. Changed.